crux-mir/report-coverage/src/main.rs

```diff
@@ -62,6 +62,10 @@ struct BlockId(pub String);
 #[derive(Clone, Debug, Default)]
 struct Report {
     fns: HashMap<FnId, FnReport>,
+    /// Only report coverage for the functions on this crates.
+    /// If this is empty, report coverage for al crates.
+    /// We use the first block in a profiling report to determine its crate.
+    report_crates: Vec<String>,
```

Comment:
Under what circumstances will `report_crates` be empty? From the code below, it looks like this will always be populated when it encounters the first block. Is there ever a situation where there are no blocks?

Reply:
At the moment it shouldn't really be empty, but I was thinking that in the future we may want to add a flag to show stats for everything, which would be conveniently implemented as just setting this field to empty.